spec/unit/content_types_spec.rb

```diff
@@ -6,9 +6,9 @@
     @klass = Jekyll::Contentful::ContentTypes
   end
 
-  it 'should return the CF space instance' do
+  it 'should return the CF environment instance' do
     VCR.use_cassette 'contentful/space' do
-      expect(@klass.send(:space)).to be_a(Contentful::Management::Space)
+      expect(@klass.send(:space)).to be_a(Contentful::Management::Environment)
```

Comment:
Reading this doesn't make the most sense but 🤷‍♂️.

`space !== env`

#unittestz